src/app/components/applications/SSV/RegisterValidatorHome/components/FundingPeriod/FundingPeriod.tsx

```diff
@@ -30,7 +30,7 @@ const FundingPeriod = () => {
   const ssvStore: SsvStore = stores.SSV;
   const processStore: ProcessStore = stores.Process;
   const operatorStore: OperatorStore = stores.Operator;
-  const [customPeriod, setCustomPeriod] = useState(730);
+  const [customPeriod, setCustomPeriod] = useState(config.GLOBAL_VARIABLE.DEFAULT_CLUSTER_PERIOD);
   const [checkedOption, setCheckedOption] = useState(options[1]);
   const timePeriodNotValid = customPeriod < 30;
```

Comment:
This also should be in a constant, like, `CLUSTER_VALIDITY_PERIOD_MINIMUM`

Reply:
done